backend/lib/src/api/routes.rs

```diff
@@ -41,18 +45,12 @@ pub fn routes(services: Services) -> Router {
             get(handlers::get_file_info),
         )
         .merge(file_upload)
+        .merge(internal_file_upload)
         .route(
             "/buckets/{bucket_id}/distribute/{file_key}",
             post(handlers::distribute_file),
         )
-        .route(
-            "/buckets/{bucket_id}/download/{file_key}",
-            get(handlers::download_by_key),
-        )
-        .route(
-            "/buckets/{bucket_id}/download/path/{*file_location}",
-            get(handlers::download_by_location),
-        )
+        .route("/files/{file_key}", get(handlers::download_by_key))
```

Comment:
I guess you removed teh `buckets/{bucket_id}` since it's not needed, right?

Reply:
yup